templates/corporate/billing.html

```diff
@@ -31,6 +31,10 @@ <h1>Zulip Cloud billing for {{ org_name }}</h1>
                             <label for="org_billing_freqency" class="inline-block label-title">Billing frequency</label>
                             <div id="org_billing_freqency" class="not-editable-realm-field">
                                 {{ billing_frequency }}
+                                {% if switch_to_annual_at_end_of_cycle %}
+                                <br />
+                                Your plan will switch to annual billing on {{ renewal_date }}.
+                                {% endif %}
```

Comment:
Should there be a parallel `switch_to_monthly_at_end_of_cycle`?

Reply:
Added to #27645 as follow up to look into. We didn't seem to have something like that in the past.